integration_test/ops_agent_test.go

```diff
@@ -24,11 +24,14 @@ to the ones mentioned at the top of gce_testing.go:
 
 PLATFORMS: a comma-separated list of distros to test, e.g. "centos-7,centos-8".
 
-The following variables are optional:
+Exactly one of the following variables must be set:
 
 REPO_SUFFIX: If provided, what package repo suffix to install the ops agent from.
```

Comment:
I think we should we do the same thing to third_party_apps_test.go so that they are consistent.